pkgs/applications/video/kodi-packages/elementum/default.nix

```diff
@@ -0,0 +1,61 @@
+{ stdenv, lib, pkgs, autoPatchelfHook, buildKodiAddon, ... }:
```

Comment:
- You can include `addonDir` in the parameters because it will get passed in.
- Instead of including `pkgs` explicitly list every package or function required.